.bazelrc

```diff
@@ -8,6 +8,10 @@ build --color=yes
 build:production --config=lsan --copt=-O3
 build:production --define=production=1
 
+# If set, then tags will be propagated from a target to the actions' execution requirements.
```

Comment:
Nit: "If set" may be confusing since it will always be set. Perhaps:
 `# experimental_allow_tags_propagation means that tags will be propagated from a target to the actions' execution requirements.`